modules/openy_features/openy_prgf/config/install/core.entity_form_display.paragraph.webform.default.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
```

Comment:
All configs that are related to new paragraph should live in separate submodule. See this as example https://github.com/ymcatwincities/openy/tree/8.x-1.x/modules/openy_features/openy_prgf/modules